changes/EXAMPLE.yaml

```diff
@@ -0,0 +1,23 @@
+# An example changelog entry
+#
+# 1. Choose one (or more if a PR encompasses multiple changes) of the following headers:
+#   - feature
+#   - enhancement
+#   - task
+#   - fix
+#   - deprecation
+#   - breaking (for breaking changes)
+#
+# 2. Fill in one (or more) bullet points under the heading, describing the change.
+#    Markdown syntax may be used.
+#
+# 3. If you would like to be credited as helping with this release, add a
+#    contributor section with your name and github username.
+#
+# Here's an example of a PR that adds an enhancement
+
+enhancement:
+  - "Example change description - [#1234](https://github.com/PrefectHQ/prefect/pull/1234)"
+
+contributor:
+  - "[Contributor Name](https://github.com/contributor_github_username)"
```

Comment:
We could definitely change this to be more structured if we wanted (for example, take in github issue/pr numbers and autogenerate the links). I went with the simplest thing possible for now.

Reply:
Example with more structure:

```yaml
description: "Example change description"
kind: enhancement
links:
  - issue: 1234
  - pr: 1235
contributor:
  name: Contributor Name
  github: contributor_username
```

Multiple changelog entries in a single file (rare) could be separated by `---`, and loaded a yaml stream.

I'm not convinced this is better, it's just an option. The current approach might be easier for those familiar with markdown (it's also simpler to implement). The more structured approach might be easier for those that aren't, and could let us enforce changelog formatting more.